we don't need all the dependencies for generating the manpage, avoids needing 4 new packages
--- a/doc/en/conf.py
+++ b/doc/en/conf.py
@@ -45,15 +45,11 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
-    "pallets_sphinx_themes",
-    "pygments_pytest",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "sphinx.ext.intersphinx",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
-    "sphinx_removed_in",
-    "sphinxcontrib_trio",
 ]
 
 # Add any paths that contain templates here, relative to this directory.
